AdminUI/src/app/app.module.ts

```diff
@@ -79,7 +80,13 @@ import { AssignQuotasDialogComponent } from './components/quotas/assign-quotas-d
         MatSelectModule,
         MatChipsModule,
     ],
-    providers: [SidebarService],
+    providers: [
+        SidebarService,
+        {
+            provide: DATE_PIPE_DEFAULT_OPTIONS,
+            useValue: { dateFormat: 'd/M/yyyy' },
```

Comment:
As the majority of the users will probably be German, can you use `dd.mm.yyyy` as the date format please? Plus: the time would be helpful as well, so you should change it to `dd.mm.yyyy HH:mm:ss`